controllers/storagecluster/storageclasses.go

```diff
@@ -292,6 +292,7 @@ func newCephBlockPoolVirtualizationStorageClassConfiguration(initData *ocsv1.Sto
 	virtualizationStorageClassConfig.storageClass.ObjectMeta.Annotations["description"] = "Provides RWO and RWX Block volumes suitable for Virtual Machine disks"
 	virtualizationStorageClassConfig.storageClass.Parameters["mounter"] = "rbd"
 	virtualizationStorageClassConfig.storageClass.Parameters["mapOptions"] = "krbd:rxbounce"
+	virtualizationStorageClassConfig.storageClass.Parameters["mapOptions"] = "krbd:osd_request_timeout"
```

Comment:
This will override the above one, please add it as `"krbd:rxbounce;krbd:osd_request_timeout"`

Reply:
Thanks, Madhu, Updated. If it looks good please LGTM. Will try to get it merged today.